### DISCUSSION TITLE
Where is the "again"-button?

### DISCUSSION THREAD

**Discussion by Superfeuchter** [st3v3nmw/obsidian-spaced-repetition#308] [Q&A] [answered]
As an Anki-User I am very thankful for your Anki-like implementation in Obsidian. It's a great time-saver.
What I unfortunately miss is a button for cards I have not answered correctly, so that the card would show up again for studying.
On my desktop I can press "0" in this case, but what can I do on my android-tablet when there ist no "again"-button?
Please give me a hint!

**Comment by cesar-programs** [comment]
Second this.

**Comment by st3v3nmw** [comment]
I've opened an issue on this.

**Comment by KyleKlus** [comment, marked as answer]
The again button was implemented a few prs ago😊
